demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -80,16 +80,17 @@ def is_existing_image(self):
 
         if data_dictionary.get('image'):
             is_image_in_yml = True
-
         if not re.match(INTEGRATION_REGEX, self.file_path, re.IGNORECASE):
             package_path = os.path.dirname(self.file_path)
-            if is_image_in_yml:
-                print_error("You have added an image in the yml "
-                            "file, please update the package {}".format(package_path))
-                return False
             image_path = glob.glob(package_path + '/*.png')
             if image_path:
                 is_image_in_package = True
+        if is_image_in_package and is_image_in_yml:
+            print_error(
+                "The file {} has image in both yml and package, remove the 'image' key from the yml file".format(
+                    self.file_path))
+            self._is_valid = False
```

Comment:
do we need to fill this in? not just return false?

Reply:
How will the user know what went wrong if we don't do this?

Reply:
im asking if by returning `False` does it not set `_is_valid` to `False` ?

Reply:
yea, missed second condition
looks good

Reply:
It doesn't set _is_valid to False